package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Themeroller",
```

Comment:
Are we planning to publish themeroller itself to npm? If not we can just add `private` here and don't add most of this whole file.